sync_issues_to_jira/README.md

```diff
@@ -183,12 +183,14 @@ The environment variables should be set in the GitHub Workflow:
 - `JIRA_ISSUE_TYPE` (optional) the JIRA issue type for new issues. If unset, "Task" is used.
 - `JIRA_COMPONENT` (optional) the name of a JIRA component to add to every issue which is synced from GitHub. The component must already exist in the JIRA project.
 
-The following secrets should be set in the workflow:
+The following secrets are needed for the workflow:
 
 - `JIRA_URL` is the main JIRA URL (doesn't have to be secret).
 - `JIRA_USER` is the JIRA username to log in with (JIRA basic auth)
 - `JIRA_PASS` is the JIRA password to log in with (JIRA basic auth)
```

Comment:
Shouldn't be?

```suggestion
- `JIRA_PASS` is JIRA token or JIRA password (JIRA basic auth) to log in with
```

Since you are using this env to store token or password in sync_issues_to_jira/sync_to_jira.py

Reply:
Thank you @tom-borcin , nice catch ... updated!